test/jasmine/tests/svg_text_utils_test.js

```diff
@@ -121,34 +121,33 @@ describe('svg+text utils', function() {
         });
 
         it('wrap XSS attacks in href', function() {
-            var node = mockTextSVGElement(
```

Comment:
reverting a commit from PR https://github.com/plotly/plotly.js/pull/804, as now `&lt;`, `&gt;` and `&quot;` are now back in play.